app/views/courses/_course.html.erb

```diff
@@ -14,7 +14,18 @@
           <h2 id="<%= course.title.parameterize %>" class="font-medium text-xl md:text-left text-gray-900 dark:text-gray-200">
             <%= course.title %>
           </h2>
-          <span class=" text-sm text-gray-500"><%= course.lessons.size %> Lessons</span>
+          <div class="flex space-x-3 mt-2 sm:mt-1.5 justify-center text-gray-500 dark:text-gray-400">
+            <div class="flex items-center space-x-1">
+              <%= inline_svg 'icons/book.svg', alt: 'lesson icon', class: 'w-6 h-6' %>
```

Comment:
Final pass through - the icons are just a little too big and draw the eye too much. 5 seems to make them more proportionate to their surroundings. 

```suggestion
              <%= inline_svg 'icons/book.svg', alt: 'lesson icon', class: 'w-5 h-5' %>
```

Size difference can be seen with this, first course has the smaller icons and the second doesnt.
<img width="946" alt="Screenshot 2024-01-24 at 14 13 09" src="https://github.com/TheOdinProject/theodinproject/assets/7963776/ae2ec106-b0d5-45f6-becb-95305ff8fffe">